### ISSUE TITLE
♿Number Input AVT1 - August 2018♿

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#1361]
## Environment
Chrome Version 70.0.3538.67 (Official Build) (64-bit)
macOS Mojave 10/Windows 10

## Violations:
Each form control must have associated label
![screen shot 2018-10-29 at 1 48 13 pm](https://user-images.githubusercontent.com/40970507/47700213-ac534e80-dbe3-11e8-900f-ac117952895f.png)

## How to reproduce
1. [Download the Dynamic Accessibility Plugin tool here](https://ibm.ent.box.com/notes/335005856253?s=mhpgjlgss354mcidsv6rvpuu0bxlmfm4)
2. Run the extension using the August 2018 ruleset

**Comment by dakahn** [comment]
I'm gonna close this. Can't seem to reproduce.
